docs/src/main/asciidoc/includes/guides/performance-tuning.adoc

```diff
@@ -28,7 +28,6 @@ Still, there might be cases where you wish to change configuration options from
 For details on the following options please see:
 
 * xref:../../se/webserver.adoc#_configuration_options[WebServer Configuration]
-* xref:../../config/io_helidon_webserver_ConnectionConfig.adoc[WebServer Connection Configuration]
 * xref:../../config/io_helidon_common_socket_SocketOptions.adoc[WebServer Socket Configuration]
 
```

Comment:
How about adding a paragraph concerning tuning for file upload performance on Linux (setting `socket-receive-buffer-size`)

Reply:
I've asked PSR to help us run a few tests on other types of environments (OCI). I think it would be nice to have that info before writing such a paragraph. Should we do that in a follow-up PR?

Reply:
Sounds good to me.